go/billing/tasks.py

```diff
@@ -384,8 +384,9 @@ def create_low_credit_notification(account_number, threshold, balance):
     # Send email
     threshold_percent = Decimal(100) - (threshold * Decimal(100))
     subject = 'Vumi Go account %s (%s) at %s%% left of available credits' % (
-        account.user.email, account.user.get_full_name(), threshold_percent)
-    email_from = settings.STATEMENT_CONTACT_DETAILS.get('email')
+        account.user.email, account.user.get_full_name(),
+        Decimal(100) - threshold_percent)
```

Comment:
I think many of the current accounts are for projects rather than for individuals, so the name and surname are a bit misued. We usually do something like `name: Girlhub, surname: Ethiopia`, since those fields are compulsory. I guess this template works for that example (it would get displayed as `(Girlhub Ethiopia)` in the subject line for that case), so maybe this is the right thing to have. I guess lets see how it goes, and if people find it looks weird, we can change it.